juego.py

```diff
@@ -128,18 +128,27 @@ def cargar_imagen(nombre, trasnparent=False):
     imagen = imagen.convert()
     return imagen
 
+def get_translated_text(text):
+    loc = locale.getdefaultlocale()[0][:2]
+    if loc not in ("en", "es"):
+        loc = "en"
+    translations = {
+        "en": [
+            "PLAY", "LEVEL", "QUIT", "easy", "medium", "hard", "Score : ", "Highest Score : ", "Timer : ","PLAY AGAIN", "GAME OVER!!", "Hurray! you won :)", "Ay! you lost :(", "Select correct ball to answer or type it using keyboard"
+        ],
+
+        "es" : [
+            "JUGAR", "NIVEL", "SALIR", "facil", "medio", "dificil", "Puntaje : ", "Puntaje Mas Alto : ", "Temporizador : ", "jUEGA DE NUEVO", "JUEGO TERMINADO!!", "Hurra! ganaste :)", "Ay! perdiste :(", "Selecciona la bola correcta para responder o escribe la respuesta usando el teclado"
+        ],
+    }
+    if text in translations["en"]:
+        return translations[loc][translations["en"].index(text)]
+    return text
```

Comment:
This is what I mean, your calls pass English words as the text hence this would always pass regardless of what the user locale is.

Reply:
Yep this will always pass if the text received is present in the English list (so its translation is known). In the case if the received text's translation is not known, we just return the text itself in English.

We can also remove this check and simply return the translation if it's known else raise error if you prefer that.

Reply:
The idea is to display Spanish when the user locale is `es` and English otherwise, currently even with a `es` locale it'll still display English because of the check and the fact that you're using English words.

The better approach would be to have a dictionary that maps English to Spanish and use that when the user locale is `es`.

Reply:
> currently even with a es locale it'll still display English

I tried what would happen in es locale by replacing:
```py
loc = locale.getdefaultlocale()[0][:2]
    if loc not in ("en", "es"):
        loc = "en"
```
with
```py
loc = "es"
```
and it shows in Spanish.
I will try your implementation though.

Reply:
I'll take your word for it as from your function that shouldn't work if locale is `es` before your implementation in ba687a2.